src/initial-html.js

```diff
@@ -4,212 +4,15 @@
  */
 
```

Comment:
Same for this, just dropping a comment to not to forget reverting this file. Let's add the new block somewhere appropriate in the demo content though.

```
<!-- wp:spacer -->
 <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>		 <div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
 <!-- /wp:spacer -->		 <!-- /wp:spacer -->
		
```